doc/release/1.8.0-notes.rst

```diff
@@ -17,7 +17,7 @@ run your code with ``python -Wd`` and check for ``DeprecationWarning`` s).
 Our development attention will now shift to bug-fix releases on the
 1.8.x branch, and on adding new features on the master branch.
 
-This release requires Python 3.8+ and NumPy 1.16.5 or greater.
+This release requires Python 3.8+ and NumPy 1.17.3 or greater.
```

Comment:
Although the minimum NumPy version is more and more like a matrix depending on your Python version and architecture these days..

Reply:
True, however I'm not sure that's worth mentioning. The people who need to know that will know where to look.